qutebrowser/browser/mouse.py

```diff
@@ -57,7 +57,29 @@ def eventFilter(self, obj, event):
 
             if qtutils.version_check('5.11', compiled=False, exact=True):
                 # WORKAROUND for https://bugreports.qt.io/browse/QTBUG-68076
-                QTimer.singleShot(0, self._widget.setFocus)
+                try:
+                    # pylint: disable=protected-access
+                    win_id = self._widget._win_id
+                    # pylint: enable=protected-access
+                    passthrough_modes = [usertypes.KeyMode.command,
+                                         usertypes.KeyMode.prompt,
+                                         usertypes.KeyMode.yesno]
+                    if modeman.instance(win_id).mode not in passthrough_modes:
+                        tabbed_browser = objreg.get('tabbed-browser',
+                                                    scope='window',
+                                                    window=win_id)
+                        current_index = tabbed_browser.widget.currentIndex()
+                        try:
+                            widget_index = tabbed_browser.widget.indexOf(
+                                self._widget.parent())
+                        except RuntimeError:
+                            widget_index = -1
+                        if current_index == widget_index:
```

Comment:
I think you could just do `if self._widget.isVisible():` instead of all this, no?

Reply:
Yah, I knew there was a better way to do this, but I didn't know what it was...

Reply:
Hmm, unfortuately that didn't work (sometimes widgets in the background have isVisible set to true). I'll have to think of something else...